### ISSUE TITLE
Fog of War is not eliminated when using the Nomad AGS

### ISSUE THREAD

**Issue by orientalsniper** [Woprok/AOE4-AdvancedGameSettings#44]
The option is activated before starting the game.

**Comment by orientalsniper** [comment]
@Woprok What's the difference between using vanilla AGS for a nomad game vs Nomad AGS?

**Comment by Woprok** [comment]
@orientalsniper There is small difference as vanilla AGS always spawns TC and mod has to remove it. Nomad AGS never spawns TC, so it doesn't need to remove it. Other difference is just reduced amount of options...

**Comment by Woprok** [comment]
Also I am not sure what is exactly the issue you are reporting. I will look into the problem some time after next patch probably.

**Comment by orientalsniper** [comment]
The issue is the option to remove fog when player is eliminated is not working. We are making a tournament and the host will surrender at minute 0 to spectate game so they can read [ALL] chat

**Comment by Woprok** [comment]
@orientalsniper any other issue or request ?

**Comment by Woprok** [comment]
Also this should be fixed now, so test it and let me know

**Comment by orientalsniper** [comment]
I can confirm issue is fixed, I don't know if I should open up a new issue, but the None option for Allied vision does not work. We want to forbid shared vision, so this is something we consider as important. This is for Nomad AGS.

**Comment by Woprok** [comment]
> I can confirm issue is fixed, I don't know if I should open up a new issue, but the None option for Allied vision does not work. We want to forbid shared vision, so this is something we consider as important. This is for Nomad AGS.

Great, I fixed the issue with Allied/Team vision None, it should now work properly. Players will still receive notification, but they will have no vision of their allies.

I really don't need more open issues :D I get nothing from solving them, other than good feeling when I can close them :D

**Comment by orientalsniper** [comment]
I can confirm None for Team Vision works, but is there a way to turn off Team Vision when a Treaty (5 mins for example) is set?

At the moment, if a treaty is set in a nomad game, everyone knows where everyone is gonna settle their TC.

**Comment by Woprok** [comment]
@orientalsniper Team Vision is currently still tied to Team Victory (it made sense when I wrote it months ago... might have not been best decision)

Team Victory -> FFA & Team Vision -> Any => Noone will have vision during treaty or during the game for any of their mutual "allies"
Team Victory -> Initial Teams & Team Vision -> Requires Market/Always => Only teams setup in lobby will share vision during the game, including treaty
Team Victory -> Dynamic Teams & Team Vision -> Requires Market/Always => everyone has vision during treaties and mutual allies see each other

**Comment by orientalsniper** [comment]
Did you make a recent change? I'm getting a downloaded mod was corrupt.

**Comment by Woprok** [comment]
Thanks for letting me know, it looks like Relic server once again failed to accept update. It should be fixed now hopefully.

**Comment by orientalsniper** [comment]
Everything works now! Did you deploy a new change to Team Vision? Even with treaty it respects No Team Vision.

Do you mind being credited in our upcoming tournament?

**Comment by Woprok** [comment]
Only fixed Team Vision None to work properly. (Bug caused it to never remove vision from players)



> Do you mind being credited in our upcoming tournament?

That would be cool.
Hopefully it will work well during the tournament. I am still not sure how stable it's compared to other Nomad mod. I have done lot of rewriting for my mod to work with all the additional options. Definitely try to run few full games to test it. There might still be few easy to trigger scar errors left in the code, which can definitely be fixed If I get some info on them.

**Comment by orientalsniper** [comment]
I must have set victory condition as to something else other than FFA, because it works now (no shared vision with all the players) with 5 minutes treaty.
